app/models/user.js

```diff
@@ -0,0 +1,36 @@
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'users',
+    {
+      id: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        unique: true,
+        autoIncrement: true,
+        primaryKey: true
+      },
+      name: {
+        type: DataTypes.STRING,
+        allowNull: false
+      },
+      last_name: {
+        type: DataTypes.STRING,
+        allowNull: false
+      },
+      email: {
+        type: DataTypes.STRING,
+        isEmail: true,
+        allowNull: false,
+        unique: true
+      },
+      password: {
+        type: DataTypes.STRING,
+        allowNull: false
+      }
+    },
+    {
+      timestamps: false
```

Comment:
You must set the option underscored: true. 
With the underscored option enabled, Sequelize will instead define a lastName attribute for user model, pointing to a column named last_name in users table.

and remember, timestamps must be set with true
Apart from this, exists paranoid option, if you set it with true, when you delete an user, a special column called deletedAt will have the timestamp of that deletion request (soft deletion)